scripts/install_ebpf.psm1

```diff
@@ -5,129 +5,60 @@ param ([Parameter(Mandatory=$True)] [string] $WorkingDirectory,
        [Parameter(Mandatory=$True)] [string] $LogFileName)
 
 Push-Location $WorkingDirectory
-
-$BinaryPath = "$Env:systemroot\system32";
-
 Import-Module $PSScriptRoot\common.psm1 -Force -ArgumentList ($LogFileName) -WarningAction SilentlyContinue
 
-# eBPF Drivers.
-$EbpfDrivers =
```

Comment:
to help review, please describe the changes in this file - there are large blocks of code removed and added - what's new and what's not?

Looks like these register function were copied into install/uninstall? Let's keep it separate dinto smaller chunks where possible

Reply:
The description of this PR covers what was introduced by this PR.